### DISCUSSION TITLE
Output Status Motion Detection DVR-SCAN

### DISCUSSION THREAD

**Discussion by fhavio** [Breakthrough/DVR-Scan#156] [Q&A]
Hi there, i'm new to dvr-scan and i'm looking for a solution that i believe is achievable.

I try the program and works very well, but it takes a long time and i just want a small motion verification. 

In my case, i'm already dividing the footage into 1h each so what i need is for dvr-scan to detect motion and report back to the script, this way i can easy mark the files that need to be sorted. 

I'm working on a cctv system and as you guys know it can grow faster than we want and it takes a while in case we need.

This way i could easily work on those of my interest, if i could also get the timeframe of movement would be great.

Any input or help is apreaciated,

Thank you.

**Comment by Breakthrough** [comment]
DVR-Scan already provides you the timecodes of each event, you use [the `-so`/`--scan-only` option](https://www.dvr-scan.com/docs/#output) if you want to skip creating files.  If you just want to see if a given video has motion, you should check the output of DVR-Scan.

I'm not sure I understand your use case fully, do you mean you want to just mark any video that has motion? If so, you can just run the command and in the output search for something like "No motion events detected in input."
